pkg/kvstore/etcd.go

```diff
@@ -515,10 +531,12 @@ func (e *etcdClient) renewLockSession(ctx context.Context) error {
 	// routines can get a lease ID of an already expired lease.
 	e.Lock()
 
+	timeoutCtx, cancel := context.WithTimeout(e.client.Ctx(), statusCheckTimeout)
```

Comment:
any reason why we can't use _ctx_ here ?

Reply:
Good catch. Fixed.